src/main/java/net/dries007/tfc/common/blocks/devices/BarrelBlock.java

```diff
@@ -90,4 +92,25 @@ protected void addExtraInfo(List<Component> tooltip, CompoundTag inventoryTag)
             tooltip.add(Tooltips.fluidUnitsOf(tank.getFluid()));
         }
     }
+
+    @Override
+    public int getAnalogOutputSignal(BlockState state, Level level, BlockPos pos)
+    {
+        return level.getBlockEntity(pos, TFCBlockEntities.BARREL.get()).map(InventoryBlockEntity::getAnalogValue).orElse(super.getAnalogOutputSignal(state, level, pos));
+    }
+
+    @Override
+    public boolean hasAnalogOutputSignal(BlockState state)
+    {
+        return true;
+    }
+
+    @Override
+    public void neighborChanged(BlockState state, Level level, BlockPos pos, Block block, BlockPos neighborPos, boolean s)
+    {
+        if (!level.isClientSide && level.hasNeighborSignal(pos))
+        {
+            toggleSeal(level, pos, state);
+        }
+    }
```

Comment:
Should barrels (which are pretty much definitionally a big pile o wood with no moving parts), actually react themselves to redstone signals? I'm on the fence for this, because I think comparator outputs are good and can be justified by "magic comparator block doing stuff" but idk, can "magic dust block doing stuff" also justify this?

@eerussianguy thoughts?

If it's contentious we could also add a config option for this doing anything (see #905)